cos/templates/game.jinja2

```diff
@@ -31,7 +31,7 @@
     <span class="row">
         <div class="col-1"></div>
         <div id="game_board" class="col-7">
-            <h1>Catan Board: Game_ID = {{game_id}}</h1>
+            <h1 id="gameName"></h1>
```

Comment:
Setting this to {{game_name}} and then setting that variable in the game_view in views.py would probably be better for this.

Reply:
OK. Will do.